### DISCUSSION TITLE
Does anyone know is CarbonSpigot have a public API that's public available?

### DISCUSSION THREAD

**Discussion by HappyAreaBean** [diamond-rip/Eden#41] [Q&A] [answered]
Does CarbonSpigot have a public API that's public available?
I want to compile Eden, but I just stuck at not having CarbonSpigot lol.

**Comment by GoodestEnglish** [comment, marked as answer]
Knockback API will be revamped with a public API along with FoxSpigot support. Would recommend to wait until that time.

**Comment by GoodestEnglish** [comment]
Update: Revamped!

**Reply by HappyAreaBean** [reply]
The revamp look awesome, but I still can't compile Eden because I don't have the spigotapi jar lol
